dns_records.php

```diff
@@ -39,6 +39,15 @@
                 </div>
             </div>
             <div class="box-footer clearfix">
+              <strong>Note:</strong>
+              <p>The order of locally defined DNS records is: </p>
+              <ol>
+                  <li>Read from the list containing the device's host name and pi.hole (stored in <code>/etc/pihole/local.list</code>)</li>
```

Comment:
```suggestion
                  <li>The device's host name and <code>pi.hole</code></li>
```

This is done inside FTL: https://github.com/pi-hole/FTL/blob/a8a2d104ea1d8d462edecc5b5158d00e10f49263/src/dnsmasq_interface.c#L403-L421

R.I.P. `local.list`